src/test/scala/chiselTests/BoringUtilsTapSpec.scala

```diff
@@ -573,6 +573,53 @@ class BoringUtilsTapSpec extends ChiselFlatSpec with ChiselRunners with Utils wi
     val verilog = circt.stage.ChiselStage.emitSystemVerilog(new Top(Definition(new Foo)))
   }
 
+  it should "work to rwTap an Instance[..]'s port" in {
```

Comment:
do we already have a test for just tap (vs rwTap)?

Reply:
I don't think so, I can add it?